mysite/polls/models.py

```diff
@@ -4,17 +4,18 @@
 from django.utils import timezone
 
 class Question(models.Model):
-    question_text = models.CharField(max_length=200)
+    question_text = models.CharField(max_length=200, unique = True)
     pub_date = models.DateTimeField('date published')
     def __str__(self):
         return self.question_text
     def was_published_recently(self):
         return self.pub_date >= timezone.now() - datetime.timedelta(days=1)
+    
 
 
 class Choice(models.Model):
     question = models.ForeignKey(Question, on_delete=models.CASCADE)
-    choice_text = models.CharField(max_length=200)
+    choice_text = models.CharField(max_length=200, unique=True)
```

Comment:
What this does is that it guarantees no two choices can have the same text regardless of the question, while what is required in this assignment is to make the choice text **unique per question**:

**Acceptable cases can be like:**
Question 1: [Choice 1, Choice 2, Choice 3]
Question 2: [Choice 1, Choice 3]

**Unacceptable case:**
Question 1 [Choice 2, Choice 2]

Reply:
Thanks for your feedback, Mahfouz. I already came up with a solution and fixed the bug and will put up a PR for that and add you as a reviewer. 

So here's my approach for solving the bug ~
I declared a Meta class within the Choice class that specifies the combination of "question" and "choice_text" must be unique together. That means no two choices with the same "choice_text" can be created for the same question.

Reply:
I tried but was not able to add you as a reviewer so I went ahead and merged it into main. 

Here's the bugfix Pull Request : https://github.com/taraamiri57/Django-Project/pull/5